zzzz_modular_occulus/code/modules/reagents/reagents/other.dm

```diff
@@ -5,4 +5,387 @@
+	if(istype(A, /mob))
+		var/mob/M = A
+		M.set_light(0)
+	return ..()
+
+/datum/reagent/liquid_skin
+	id = "liquid_skin"
+	name = "Liquid Skin"
+	description = "Fills in microscopic ridges on biotic surfaces and hardens"
+	color = "#F7E9BE"
+
+/datum/reagent/liquid_skin/touch_mob(var/mob/M, var/volume)
+	if(istype(M, /mob/living))
+		var/mob/living/L = M
+		var/burned = L.getFireLoss() > 0
+		if(burned)
+			L << "<span class='notice'>In a strange sensation, you feel some burns stop hurting.</span>"
+			L.heal_organ_damage(0, min(15, volume / 4))
+
+		if (mFingerprints in L.mutations)
+			if(!burned)
+				L << "<span class='warning'>Another application of the substance does nothing weird to your hands.</span>"
+		else
+			L.mutations.Add(mFingerprints)
+			L << "<span class='notice'>Your fingers feel strange after the substance splashes on your hands.</span>"
+	return ..()
+
+/datum/reagent/paralitol
+	id = "paralitol"
+	name = "Paralitol"
+	description = "Seems as if it could work as an extreme muscle inhibitor"
+	color = "#2F85CC"
+	metabolism = 3 * REM
+
+/datum/reagent/paralitol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	M.Weaken(2)
+	M.stuttering = 50
+	return ..()
+
+/datum/reagent/mortemol
+	id = "mortemol"
+	name = "Mortemol"
+	description = "Further testing required, could potentially reanimate dead cells if delivered with enough force"
+	color = "#000000"
+	metabolism = 5 * REM //gotta balance it somehow
+	data = list(0) //use data? Might cause problems with blood dialysis
+
+/datum/reagent/mortemol/touch_mob(var/mob/M, var/volume) //requires a splash to start effects because dead humans don't process reagents
+	if(!istype(M, /mob/living/carbon))
+		return 0
+
+	var/mob/living/carbon/C = M
+	if(holder)
+		if(!istype(holder.my_atom, /obj/effect/effect/smoke/chem))
+			if(C.reagents)
+				if(C.stat && !(data[1]))
+					data[1] = 1
+					C.reagents.add_reagent(id, volume, data)
+					C.rejuvenate()
+					C.rejuvenate() //I like C.rejuvenate()
+					C.visible_message("<span class='notice'>[C] seems to wake from the dead!</span>")
+				else
+					C.reagents.add_reagent(id, volume)
+
+/datum/reagent/mortemol/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
+	if(data[1])
+		M.halloss = 100
+		M.stuttering = 1
+
+	return ..()
+
+/datum/reagent/mortemol/on_remove(var/atom/A)
+	if(data[1])
+		if(istype(A, /mob))
+			var/mob/M = A
+			to_chat(M, "<span class='notice'>You feel the last traces of chemicals leave your body as you return to death once more...</span>")
+			M.death(0)
+		//Reagent giveth, and reagent taketh away
+
+/datum/reagent/mortemol/on_mob_death(var/mob/M)
+	if(data[1])
+		return
+	else
+		if(istype(M, /mob/living/carbon))
+			var/mob/living/carbon/C = M
+			data[1] = 1
+			C.rejuvenate()
+			C.rejuvenate() //Necessary to call twice in testing
+			C.visible_message("<span class='notice'>[C] seems to wake from the dead!</span>")
+
+/datum/reagent/oxyphoromin
```

Comment:
This is overkill and unnecessary. Oxy is already overkill

Reply:
Has been yeeted. Looking at finding a replacement chem for it atm. Probably the one you suggested related to bluespace dust